CONTRIBUTING.md

```diff
@@ -24,7 +24,9 @@ For more information, please see STYLEGUIDE.md
 
 # <a name="Attachments"></a>Attachments
 
-GitHub only allows image attachments. Other attachments, for example zipped logs, can be posted in the <a href="http://www.universalmediaserver.com/forum/viewtopic.php?f=14&t=1656">Github attachments thread</a> in the UMS forum.
+GitHub only allows few attachments formats. Other attachments, for example zipped logs, can be posted in the <a href="http://www.universalmediaserver.com/forum/viewtopic.php?f=14&t=1656">Github attachments thread</a> in the UMS forum.
```

Comment:
"GitHub only allows **a** few attachments **extensions**: ```.png```, ```.gif```, ```.jpg```, ```.docx```, ```.pptx```, ```.xlsx```, ```.txt```, ```.pdf```, ```.zip``` and ```.gz```. (GitHub doesn't check the format at all). Reference: https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/

Remove ```, for example zipped logs```.